### PR TITLE
Fix RAILS_VERSION echo lines and add command to drop db if exists

### PR DESCRIPTION
**Pull request by klgilbert**

### CODE REVIEW

File: rails/rails_releases/rails_benchmarks/runner [hunk 1]
```diff
@@ -11,18 +11,21 @@ while ! exec 6<>/dev/tcp/${MYSQL_PORT_3306_TCP_ADDR}/${MYSQL_PORT_3306_TCP_PORT}
...
+dropdb --if-exists -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
 createdb -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
+echo "DROP DATABASE IF EXISTS rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
 echo "CREATE DATABASE rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
 
 cd /ruby-bench-suite/rails
 git pull --rebase origin master
 
-echo 'Using Rails $RAILS_VERSION'
+echo "Using Rails $RAILS_VERSION"
```
Comment (tgxworld) [inline rails/rails_releases/rails_benchmarks/runner:23]: :+1:  Thanks!

File: rails/rails_releases/rails_benchmarks/runner [hunk 2]
```diff
@@ -11,18 +11,21 @@ while ! exec 6<>/dev/tcp/${MYSQL_PORT_3306_TCP_ADDR}/${MYSQL_PORT_3306_TCP_PORT}
   sleep 1
 done
 
+
+dropdb --if-exists -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
 createdb -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
+echo "DROP DATABASE IF EXISTS rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
```
Comment (tgxworld) [inline rails/rails_releases/rails_benchmarks/runner:17]: Any reason we need to drop the databases? The scripts are run against a new PostgreSQL/MySQL container each time

**Comment by klgilbert** [comment]
I've been running it locally and I don't create new containers for the databases every time. I wrote a script to run the the benchmarks for the different releases of Rails. It's not necessary if you create new containers.

File: rails/rails_releases/rails_benchmarks/runner (hunk 2 above)
Comment (klgilbert) [inline rails/rails_releases/rails_benchmarks/runner:17]: I've been running it locally and I don't create new containers for the databases every time. I wrote a script to run all the the benchmarks for the different releases of Rails. It's not necessary if you create new containers.

Comment (tgxworld) [inline rails/rails_releases/rails_benchmarks/runner:17]: Ah OK. You're running them in a script. Sounds good :)

**Pull request merged**